spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -158,19 +158,19 @@ def after_hooks(source)
             end
           end
 
-          expect( formatter.legacy_messages ).to eq [
-              :before_features,
-                :before_feature,
+          expect(formatter.legacy_messages).to eq [
+            :before_features,
+              :before_feature,
+                :before_tags,
+                :after_tags,
+                :feature_name,
+                :before_feature_element,
                   :before_tags,
-                  :after_tags,
-                  :feature_name,
-                  :before_feature_element,
-                    :before_tags,
-                      :after_tags,
-                    :scenario_name,
-                  :after_feature_element,
-                :after_feature,
-              :after_features,
+                    :after_tags,
+                  :scenario_name,
+                :after_feature_element,
+              :after_feature,
+            :after_features,
           ]
```

Comment:
Hi @pmatsinopoulos, sorry for the late reply. Makes sense! It would be great to fix the indentation as well. I that all the elements of the array should be aligned with an indentation of 2 spaces. What do you think?